src/curve.rs

```diff
@@ -957,6 +957,30 @@ impl ExtendedPoint {
     }
 }
 
+/// Holds odd multiples 1A, 3A, ..., 15A of a point A.
+struct OddMultiples(pub [ProjectiveNielsPoint; 8]);
```

Comment:
Nope, thanks, I made it pub to have literal constants -- but constants should be affine, not projective points. I'll fix it.